### DISCUSSION TITLE
Which helm, kubectl versions are compatible with rke2 1.24.0 ?

### DISCUSSION THREAD

**Discussion by mtgmmansour** [rancher/rke2#3935] [Q&A] [answered]
The latest version of helm client is currently: v3.11.1 and the latest version of kubectl is v1.24.0

How do I know which client versions are compatible with every rke2 release?

**Comment by brandond** [comment, marked as answer]
You should be looking in the helm and Kubernetes docs. The version skew policies are pretty easy to find:

* https://helm.sh/docs/topics/version_skew/
* https://kubernetes.io/releases/version-skew-policy/#kubectl
